src/app/login/login.component.html

```diff
@@ -63,8 +66,9 @@ <h1 i18n>Login</h1>
             >Log in</button
           >
         </form>
+      </div>
         <ng-container *ngIf="socialApps$ | async as socialApps">
-          <div *ngIf="socialApps.length" class="or">
+          <div *ngIf="socialApps.length > 1" class="or">
```

Comment:
IS this the correct way to do this  in Angular `<div *ngIf="socialApps.length > 1" class="or">`?

Reply:
@vinaybommana is there somewhere I can see this working?